qa/rpc-tests/sc_cert_nonceasing.py

```diff
@@ -43,13 +43,6 @@ def setup_network(self, split=False):
         sync_mempools(self.nodes[1:NUMB_OF_NODES])
         self.sync_all()
 
-    def split_network(self):
-        # Split the network of three nodes into nodes 0 and 1
-        assert not self.is_network_split
-        disconnect_nodes(self.nodes[0], 1)
-        disconnect_nodes(self.nodes[1], 0)
-        self.is_network_split = True
-
     def join_network(self):
```

Comment:
`join_network` can be removed and the call at line 316 modified inserting parameter `id=0` (if the `time.sleep` call is really mandatory, it can be inserted just after, at line 317).